File: planner.cpp
#include <iostream>
#include <random>
#include <vector>
#include "Objects.h"
#include "planner.h"

void main() {
	RRT_Planner planner;
	Node newNode;
	SquareObject object(90, 50 ,50);
	if(planner.solveQuery(1000,1,2,2,98,98,object))
	{	
		std::cout << std::endl << "SUCCESS" << std::endl;
	}
	system("PAUSE");
}

File: planner.java
package oopProject;

//import oopa.SquareObject;

public class planner {

	public static void main(String[] args) {
		RRT_Planner planner= new RRT_Planner();
		//Node newNode=new Node();
		SquareObject object= new SquareObject(90, 50, 50);
		if (planner.solveQuery(2000, 1, 2, 2, 98, 98, object))
		{
			System.out.println("SUCCESS" );
		}
		
	}
	
	
}
